### PR TITLE
move icons

### PR DESCRIPTION
**Pull request by webmsgr**
Ooh new icons

### CODE REVIEW

**Comment by webmsgr** [comment]
Please check that it looks ok on iOS devices

File: src/pwa/manifest.webmanifest [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      },
+      {
+        "src": "./icons/icon.png",
+        "type": "image/png",
+        "sizes": "512x512"
+      }
+    ],
+    "start_url": "/?source=pwa",
+    "background_color": "#272727",
+    "display": "fullscreen",
```
Comment (insberr) [inline src/pwa/manifest.webmanifest:29]: should be "standalone"

**Review by insberr — CHANGES_REQUESTED** [review]
display should be "standalone" not "fullscreen"

**Comment by webmsgr** [comment]
you did it so you gotta accept the changes man

**Review by insberr — APPROVED** [review]

**Pull request merged**